Makefile

```diff
@@ -9,7 +9,11 @@ build-image:
 	docker build -t sigdocs -f generator/Dockerfile generator
 
 gen-doc:
-	docker run -e WG=${WG} -e SIG=${SIG} -v $(shell pwd):/go/src/app sigdocs
+	docker run -e WG -e SIG -v $(shell pwd):/go/src/app sigdocs
 
 gen-docs:
-	docker run -v $(shell pwd):/go/src/app sigdocs
+	docker run -v $(shell pwd):/go/src/app/generated sigdocs
+
+test-unit:
+	docker build -t sigdocs-test -f generator/Dockerfile.test generator
```

Comment:
I'm also wondering if we actually need a separate image for testing, or if we could use the same docs image, but then override the CMD from docker run.